devel.json

```diff
@@ -58,6 +58,17 @@
     },
     "ibm_cloud": {
       "accounts": ["my_ic_account_one", "my_ic_account_two"]
+    },
+    "kubernetes": {
+      "cluster_resources": {
+        "clusters": [
+          {
+            "account": "myaccount1",
+            "server": "https://myserver1:30000"
+          }
+        ],
+        "types": ["nodes", "pods", "apigroup.example.com/v1/mycustom"]
+      }
     }
```

Comment:
This doesn't match your README write up.  I think a test could have caught that.

Reply:
updated devel.json by copying config data from my local unittest file with anonymization.